test/rook/start

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import sys
+
+import drenv
+
+# Update this when upgrading rook.
+ROOK_BASE_URL = "https://raw.githubusercontent.com/rook/rook/release-1.10/deploy/examples"
```

Comment:
(future) Is it possible to configure per module args from the higher level input YAML file (like `test/regional-dr.yaml`). The intention being, we could run different versions of Rook or other dependent components across different test. 

With the current scheme each modules args would need to be edited.

Reply:
Good idea, I think we need to make this much more declarative as we go.